src/stylesheets/livefyre_widget_overrides.scss

```diff
@@ -469,9 +469,9 @@
 		}
 
 		.fyre-comment-flag-mobile-btn {
-			
+
```

Comment:
Ah ok, sorry was commenting on the newline here, should probably not be a line here no?